fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/typing/Type.java

```diff
@@ -1462,6 +1517,8 @@ public void defineProtoType(final TypeRepository.Builder typeRepositoryBuilder)
                     FieldDescriptorProto.Label.LABEL_OPTIONAL);
             typeRepositoryBuilder.addMessageType(helperDescriptorBuilder.build());
             typeRepositoryBuilder.registerTypeToTypeNameMapping(this, typeName);
+
+             */
```

Comment:
so I'm a little confused in this part:
1) before the array wrapper, why do we add a "value" field in this method?
2) maybe I should put what is in `addProtoField` in here? right now the two methods seem to be calling each other which makes me confusing.